lib/helpers.js

```diff
@@ -123,6 +125,17 @@ export const parseResults = (editor, results, filePath, showIgnored) => {
   return toReturn;
 };
 
+export const saveCleanedStyles = async (editor, results) => {
```

Comment:
Let's rename this to `setFixedStyles` or something along those lines, this isn't saving the file (nor should it in my opinion).